agent/Makefile.frag

```diff
@@ -49,13 +49,19 @@ $(PHP_MODULES): .libs/deps.mk
 newrelic.la: $(PHP_AXIOM)/libaxiom.a
 
 #
-# The version number is needed by php_newrelic.c as a static string literal,
+# The version number is needed by several source files as a static string literal,
 # so it can be placed in the module entry.
 #
 include ../make/version.mk
+php_api_internal.lo: CPPFLAGS += -DNR_VERSION="\"$(AGENT_VERSION)\""
+php_api_internal.lo: ../VERSION
+
```

Comment:
Should be reverted as part of 7ea6c93be430eeadd4671cc76923a6b17e78c5f8

Reply:
Good catch - fixed in [fe93413c](https://github.com/newrelic/newrelic-php-agent/pull/947/commits/fe93413c374d0700f79d6122d336f31f2c3383b1).